### PR TITLE
Git/chrome ext

### PR DESCRIPTION
**Pull request by the-thief**
Hi Belugas,

I added two Git sheet sheets in PDF format as well as the URLs of these files to the git page. They are similar but have different content. They are quite handy for common git commands.

* [Git Cheat Sheet - Education](https://education.github.com/git-cheat-sheet-education.pdf)
* [Git Cheat Sheet](https://services.github.com/on-demand/downloads/github-git-cheat-sheet.pdf)

I also create a new Chrome Extensions page and added a Youtube video I found on the CS50 channel by Walter Martin about Chrom Extensions. I'm sure it will come in handy for those that are working on the Momentum project. I wasn't sure about the category so I made a new page under the learn folder.

* https://youtu.be/Zz7HRDj9E-o

### CODE REVIEW

**Comment by the-thief** [comment]
Sounds good to me. There could be a problem is someone goes looking for
Chrome resources.

Can we do both on this one?

I want to add some Chrome dev tools/debugging stuff later.

Let me know what you think.

Thanks.

On Feb 23, 2017 11:23 PM, "Gordana Minovska" <notifications@github.com>
wrote:

*@gminovska* commented on this pull request.

@the-thief <https://github.com/the-thief> I think we should rename the file
to cs50 instead of chrome-ext, I expect we will have more resources
pertinent to this course. What are your thoughts on this?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/AgedLace/chingu-belugas-resource-library/pull/59#pullrequestreview-23647421>,
or mute the thread

File: learn/git-github.md [hunk 1]
```diff
@@ -7,3 +7,5 @@
 * Lynda.com [Github for Web Designers](https://www.lynda.com/GitHub-tutorials/GitHub-Web-Designers/162276-2.html)
 * [Common Git Workflows](https://www.atlassian.com/git/tutorials/comparing-workflows)
 * [Help I messed up!](http://sethrobertson.github.io/GitFixUm/fixup.html)
+* [Git Cheat Sheet - Education](https://education.github.com/git-cheat-sheet-education.pdf)
```
Comment (gminovska) [inline learn/git-github.md]: These two are great! I am going to print them and have them on my desk. Thank you!

**Comment by gminovska** [comment]
@the-thief Agreed, If you will add more resources for Chrome than it makes sense to have it as a separate category.

**Comment by the-thief** [comment]
I'll put a couple of things together and add them. Hopefully by the weekend.

**Pull request merged**